platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -12,40 +12,41 @@ MGL_EXPORT
 @interface MGLMapSnapshotOptions : NSObject
 
 /**
- Creates a set of options with the minimum required information
+ Creates a set of options with the minimum required information.
+ 
  @param styleURL the style url to use
  @param camera the camera settings
  @param size the image size
  */
 - (instancetype)initWithStyleURL:(NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
```

Comment:
`styleURL` should be marked nullable, given the change to the implementation.

/ref https://github.com/mapbox/mapbox-gl-native/pull/9891#discussion_r139851382